<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="15" started-at="2019-03-06T10:16:42Z" finished-at="2019-03-06T10:16:42Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="15" started-at="2019-03-06T10:16:42Z" finished-at="2019-03-06T10:16:42Z">
      <class name="newThings.TestNGDataProvider">
        <test-method status="SKIP" signature="test(java.lang.String, java.lang.String)[pri:0, instance:newThings.TestNGDataProvider@411f53a0]" name="test" duration-ms="0" started-at="2019-03-06T15:46:42Z" finished-at="2019-03-06T15:46:42Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method test requires 2 parameters but 0 were supplied in the @Test annotation.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method test requires 2 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:193)
	at org.testng.internal.Parameters.createParameters(Parameters.java:129)
	at org.testng.internal.Parameters.createParameters(Parameters.java:378)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:488)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1240)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- newThings.TestNGDataProvider -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
